coalib/coala.py

```diff
@@ -69,15 +72,24 @@ def main():
             return 0
 
     except BaseException as exception:  # pylint: disable=broad-except
+        if not isinstance(exception, SystemExit):
+            if debug:  # pragma: no cover
```

Comment:
@sils Any idea of how to write some proper fail test that raises something else than `SystemExit` here? To get the `# pragma` away